handlebars/helpers/escape.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function(variable) {
+    // return variable.replace(/(['"])/g, '\\$1');
+    
+    return variable.replace(/[\s\S]/g, function(character) {
```

Comment:
Why `/[\s\S]/g` instead of `/./g`?

Reply:
Good catch. I missed that when copying this over from StackOverflow.